SwiftLeeds/Views/Local/LocalViewModel.swift

```diff
@@ -8,23 +8,35 @@
 import Foundation
 import NetworkKit
 import SwiftUI
+import MapKit
 
 class LocalViewModel: ObservableObject {
+    @Published var selectedCategory: Local.LocationCategory?
+    @Published var categories: [Local.LocationCategory] = []
+
     @Environment(\.network) var network: Networking
 
-    var categories: [Local.LocationCategory] = []
     var error: Error?
     
     func loadData() async {
         do {
             let localResults = try await network.performRequest(endpoint: LocalEndpoint())
-            self.categories = localResults.data
+
+            DispatchQueue.main.async {
+                self.categories = localResults.data
+                self.selectedCategory = self.categories.first
+            }
         } catch {
             self.error = error
         }
+
         await publishResults()
```

Comment:
This `publishResults` was only here because I wasn't using `Published`, leaving this in alongside `Published` will cause a third update which isn't needed

The `self.categories = localResults.data` will already cause the UI to update due to the property wrapper, and same for `self.selectedCategory = self.categories.first`

Background on this area as to why/how I did it the other way here: 

https://www.hackingwithswift.com/quick-start/swiftui/how-to-send-state-updates-manually-using-objectwillchange

TLDR `@Published` _or_ `objectWillChange.send()`, but generally not both

Reply:
Remove redundant objectWillChange.send()